testsupport/init.go

```diff
@@ -67,6 +69,36 @@ func waitForOperators(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	cl.Create(context.TODO(), &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "e2e_test",
```

Comment:
I don't think that underscore is a valid char in k8s names

Reply:
yeah ..  changed it now